src/System.IO.FileSystem/src/System/IO/UnixFileSystem.cs

```diff
@@ -653,5 +653,10 @@ public override IFileSystemObject GetFileSystemInfo(string fullPath, bool asDire
                 (IFileSystemObject)new DirectoryInfo(fullPath, null) :
                 (IFileSystemObject)new FileInfo(fullPath, null);
         }
+
+        public override string[] GetLogicalDrives()
+        {
+            return DriveInfoInternal.GetLogicalDrives();
```

Comment:
> I need to use the same exact code in System.Environment

You won't be able to do that currently: Environment is in coreclr, not corefx.  See https://github.com/dotnet/coreclr/pull/6006.

> I know it's one line, but it still would be duplicated

I get that, but there's going to be duplication regardless.  If there are two calls sites, either you have:
```C#
// call site 1
return Interop.Sys.GetAllMountPoints().ToArray();

// call site 2
return Interop.Sys.GetAllMountPoints().ToArray();
```
or you have:
```C#
// call site 1
return DriveInfoInternal.GetLogicalDrives();

// call site 2
return DriveInfoInternal.GetLogicalDrives();

// Separate file containing helper implementation
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.
// See the LICENSE file in the project root for more information.

using System.Diagnostics;
using System.Text;

namespace System.IO
{
    /// <summary>Contains internal volume helpers that are shared between many projects.</summary>
    internal static partial class DriveInfoInternal
    {
        internal static string[] GetLogicalDrives()
        {
            return Interop.Sys.GetAllMountPoints().ToArray();
        }
    }
}
```
I'm not seeing how the latter is better than the former.